src/Apps/CollectorProfile/Routes/Saves2/Components/Actions/EditSavesModal.tsx

```diff
@@ -0,0 +1,177 @@
+import {
+  Button,
+  Flex,
+  LabeledInput,
+  ModalDialog,
+  Spacer,
+  Text,
+  useToasts,
+} from "@artsy/palette"
+import EditIcon from "@artsy/icons/EditIcon"
+import { useTranslation } from "react-i18next"
+import { SavesArtworks_collection$data } from "__generated__/SavesArtworks_collection.graphql"
+import { Form, Formik, FormikHelpers } from "formik"
+import * as Yup from "yup"
+import { pick } from "lodash"
+import { useMutation } from "Utils/Hooks/useMutation"
+import { graphql } from "react-relay"
+import { EditSavesModalMutation } from "__generated__/EditSavesModalMutation.graphql"
+import createLogger from "Utils/logger"
+
+interface Props {
```

Comment:
This is #superminor - since props map directly to the component name, might as well keep things 1:1 -- `EditSavesModalProps`. then searches for `EditSavesModal` yield searches for the other and it remains simpler.